app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, current_file, new_file = command.split()
+    if (current_file != new_file
+            and command == "cp" and (current_file, new_file)):
```

Comment:
```and (current_file, new_file)):```

Can you explain why did you use this construction and how does this work?

Reply:
current_file is True, new_file is True I mean